packages/reverse-exports/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "reverse-exports",
+  "version": "0.0.0",
+  "description": "",
+  "main": "src/index.js",
+  "scripts": {
+    "test": "jest"
```

Comment:
by what magic does this (`jest`) not need to be in dev deps?

Reply:
it's in the monorepo deps, it's the same way that the other packages that use jest do things